### PR TITLE
2020 12 update

### PR DESCRIPTION
**Pull request by Relequestual**
This SHOULD be all the changes required for 2020-12.

There should be a build preview appearing just below this comment.

I've also added local build instructions to the readme should you want to run it locally to test any changes.

### CODE REVIEW

**Comment by karenetheridge** [comment]
I'm not sure if these are problems, or just weirdness with the preview:


- Understanding JSON Schema is a 404 on https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/,

- the metaschema and vocabulary links to go a text file with content like "../../../_includes/draft/2020-12/meta/core.json" on https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/specification.html
and
https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/specification-links.html#2020-12

**Comment by jdesrosiers** [comment]
All of the schema examples under `/learn` have a `$schema` set as draft-07. Those should all be updated to the latest draft.

File: index.md [hunk 1]
```diff
@@ -22,43 +20,56 @@ permalink: /
...
+Luckily, we created this [getting started guide](/learn/getting-started-step-by-step)!
 
-</div>
+After that, you could check out all our [learning resources](/learn).
+
+### Got questions?
+
+We're here to help!
+
+At any point, feel free to join our Slack server using the link in the menu.
```
Comment (jdesrosiers) [inline index.md:35]: Why not just add the link here as well.

**Comment by jdesrosiers** [comment]
When we posted the draft 2020-12 review in slack, there was some confusion about Relative JSON Pointer's role in JSON Schema. It might be prudent to address that confusion when we publish the website. A note on the specifications page should be enough.

**Comment by Relequestual** [comment]
> I'm not sure if these are problems, or just weirdness with the preview:
> 
>     * Understanding JSON Schema is a 404 on https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/,

That's fine. That path specifically is published under the other repo. It's weird but managed by github settings and it works!

> 
>     * the metaschema and vocabulary links to go a text file with content like "../../../_includes/draft/2020-12/meta/core.json" on https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/specification.html
>       and
>       https://deploy-preview-369--condescending-hopper-c3ed30.netlify.app/specification-links.html#2020-12

I may need to update submodules, or I forgot to include the update for submodules. Checking.

**Comment by Relequestual** [comment]
> When we posted the draft 2020-12 review in slack, there was some confusion about Relative JSON Pointer's role in JSON Schema. It might be prudent to address that confusion when we publish the website. A note on the specifications page should be enough.

@jdesrosiers I'm going to make a new issue for this as I'm not really sure what you had in mind, and I don't want to delay this PR any longer than required.

File: specification.md [hunk 2]
```diff
@@ -5,71 +5,72 @@ permalink: /specification.html
...
 
-The current version is *2019-09*!
+The current version is *2020-12*!
+The previous version was 2019-09.
 
 Specification documents
 -----------------------
 
-See also the [release notes / change log](draft/2019-09/release-notes.html).
+See also the release notes / change log (Work in progress).
```
Comment (philsturgeon) [inline specification.md:14]: Lets link to the page, it alread ysays its a work in progress when they get there. One less thing to remember to update.

**Review by philsturgeon — APPROVED** [review]
This looks good, with the OAS3.1 announcement it would be good to get this up ASAP.

File: specification.md (hunk 2 above)
Comment (Relequestual) [reply specification.md:14]: Good plan. I can't make the change right now, but feel free to make a change suggestion =]

Comment (philsturgeon) [reply specification.md:14]: Eh skip it and merge, this is good.

**Comment by philsturgeon** [comment]
Fixed the conflict, let's press on and make more improvements shortly.

**Comment by jdesrosiers** [comment]
> All of the schema examples under /learn have a $schema set as draft-07. Those should all be updated to the latest draft.

This hasn't been addressed yet. I send a PR shortly.

**Comment by Relequestual** [comment]
> 
> 
> > All of the schema examples under /learn have a $schema set as draft-07. Those should all be updated to the latest draft.
> 
> This hasn't been addressed yet. I'll send a PR shortly.
- @jdesrosiers 

Given it hasn't been updated since draft-07, and we've had 2019-09 since then, I'm going to say this should be covered by the load of docs work that needs to be done, not in this PR specifically.

**Pull request merged**